demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -584,42 +574,37 @@ static const char * pPath;
 /**
  * @brief Update File path buffer.
  */
-static uint8_t updateFilePath[ OTA_MAX_FILE_PATH_SIZE ];
+static uint8_t ucUpdateFilePath[ OTA_MAX_FILE_PATH_SIZE ];
 
 /**
  * @brief Certificate File path buffer.
  */
-static uint8_t certFilePath[ OTA_MAX_FILE_PATH_SIZE ];
-
-/**
- * @brief Stream name buffer.
- */
-static uint8_t streamName[ OTA_MAX_STREAM_NAME_SIZE ];
+static uint8_t ucCertFilePath[ OTA_MAX_FILE_PATH_SIZE ];
 
 /**
  * @brief Decode memory.
  */
-static uint8_t decodeMem[ otaconfigFILE_BLOCK_SIZE ];
+static uint8_t ucDecodeMem[ otaconfigFILE_BLOCK_SIZE ];
 
 /**
  * @brief Bitmap memory.
  */
-static uint8_t bitmap[ OTA_MAX_BLOCK_BITMAP_SIZE ];
+static uint8_t ucBitmap[ OTA_MAX_BLOCK_BITMAP_SIZE ];
 
 /**
  * @brief Certificate File path buffer.
  */
-static uint8_t updateUrl[ OTA_MAX_URL_SIZE ];
+static uint8_t ucUpdateUrl[ OTA_MAX_URL_SIZE ];
 
 /**
  * @brief Auth scheme buffer.
  */
-static uint8_t authScheme[ OTA_MAX_URL_SIZE ];
```

Comment:
Updated prefixes, `otaexample` instead of `OTA_`  in https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724